### PR TITLE
Recookiecut

### PR DESCRIPTION
**Pull request by bobleesj**

### CODE REVIEW

**Comment by bobleesj** [comment]
(CI running)

**Comment by bobleesj** [comment]
Ready for review @sbillinge

File: .gitignore [hunk 1]
```diff
@@ -93,7 +93,7 @@ target/
 
 # version information
 setup.cfg
-/src/bg-mpl-stylesheets/*/version.cfg
+/src/billingegroup/*/version.cfg
```
Comment (sbillinge) [inline .gitignore:96]: this seems to be wrong?  was correct before?

File: doc/source/index.rst [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 .. |title| replace:: bg-mpl-stylesheets documentation
 
-bg-mpl-stylesheets - A package for using Billinge group style files.
+bg-mpl-stylesheets - A package for using Billinge group style files..
```
Comment (sbillinge) [inline doc/source/index.rst:7]: extra period?

Comment (sbillinge) [reply doc/source/index.rst:7]: also, while we are at it, let's say "Matplotlib style files" instead of just style files

**Review by sbillinge — COMMENTED** [review]
maybe a couple of things here.  Please see inline

File: doc/source/index.rst (hunk 2 above)
Comment (bobleesj) [reply doc/source/index.rst]: done

File: .gitignore (hunk 1 above)
Comment (bobleesj) [reply .gitignore]: done

File: .gitignore [hunk 3]
```diff
@@ -93,7 +93,7 @@ target/
 
 # version information
 setup.cfg
-/src/bg-mpl-stylesheets/*/version.cfg
+/src/bg_mpl_stylesheets/*/version.cfg
```
Comment (bobleesj) [inline .gitignore:96]: snakecase

File: doc/source/index.rst [hunk 4]
```diff
@@ -4,7 +4,7 @@
 
 .. |title| replace:: bg-mpl-stylesheets documentation
 
-bg-mpl-stylesheets - A package for using Billinge group style files.
+bg-mpl-stylesheets - A package for using Billinge group Matplotlib style files.
```
Comment (bobleesj) [inline doc/source/index.rst:7]: extra "." gone, Matplotlib style added

File: pyproject.toml [hunk 5]
```diff
@@ -11,7 +11,7 @@ authors = [
 maintainers = [
   { name="Simon J.L. Billinge group", email="simon.billinge@gmail.com" },
 ]
-description = "A package for using Billinge group style files"
+description = "A package for using Billinge group Matplotlib style files."
```
Comment (bobleesj) [inline pyproject.toml:14]: added "Matplotlib" as well to pyproject.toml

**Comment by bobleesj** [comment]
Ready for review again @sbillinge

File: doc/source/api/bg-mpl-stylesheets.rst [hunk 6]
```diff
@@ -0,0 +1,44 @@
+:tocdepth: -1
+
+bg-mpl-stylesheets package
```
Comment (sbillinge) [inline doc/source/api/bg-mpl-stylesheets.rst:3]: please check.  Should these be underscores here? snake_case?

**Review by sbillinge — COMMENTED** [review]
one last ;thing to check.

S

File: doc/source/api/bg-mpl-stylesheets.rst (hunk 6 above)
Comment (bobleesj) [reply doc/source/api/bg-mpl-stylesheets.rst:3]: noted, will double check this tmr

**Pull request merged**